src/telemetry/EUDomainNames.ts

```diff
@@ -3,7 +3,8 @@ const EUDomainNames = [
     "crm12.omnichannelengagementhub.com", // FRA
     "crm16.omnichannelengagementhub.com", // GER
     "crm17.omnichannelengagementhub.com", // CHE
-    "crm19.omnichannelengagementhub.com" // NOR
+    "crm19.omnichannelengagementhub.com", // NOR
+    "eu.omnichannelengagementhub.com" // PACS

```

Comment:
I think we need to add respective PACS domain for these

"crm4.omnichannelengagementhub.com", // EUR
    "crm12.omnichannelengagementhub.com", // FRA
    "crm16.omnichannelengagementhub.com", // GER
    "crm17.omnichannelengagementhub.com", // CHE
    "crm19.omnichannelengagementhub.com", // NOR